### DISCUSSION TITLE
How to abbreviate the content of cite, such as [1,2,3,4]->[1-4]

### DISCUSSION THREAD

**Discussion by clq9920** [typst/typst#2050] [Q&A] [answered]
x: #cite("a2000","b2000","c2000")->[1, 2, 3]
o: #cite("a2000","b2000","c2000")->[1- 3]

**Comment by yochem** [comment, marked as answer]
Use the following:
```typst
#cite(<a2000>)#cite(<b2000>)#cite(<c2000>)
bibliography("refs.bib", style: "springer-lecture-notes-in-computer-science")
```
Or any other style that supports range citations.

**Reply by yochem** [reply]
If you don't want to type `#cite(<a2000>)#cite(<b2000>)#cite(<c2000>)`, but your shorter form, use this:

```typst
#let multicite(keys) = {
  for key in keys {
    cite(key)
  }
}

#multicite((<alpaydin2014>,<anderson2015>))
```
Note the double parentheses. These are necessary.

**Reply by laurmaedje** [reply]
`@alpaydin2014 @anderson2015` is the intended way. There are few minor issues with it (with trailing spaces), but those will be fixed in 0.10. While citation grouping is already supported this way, the range citations will also only ship in 0.10.
